src/View/User/_addUserForm.html.twig

```diff
@@ -0,0 +1,79 @@
+{% block stylesheet %}
+<link rel="stylesheet" href="/assets/css/userForm.css">
+{% endblock %}
+
+{% block content %}
+	 <section class="formSection">
+	 <form method="post" enctype="multipart/form-data" action="users/add">
+
+		<div class="image-holder" id="image-holder">
+		<img src="https://www.w3schools.com/howto/img_avatar.png" alt="Avatar" class="userPic" >
+		</div>
+
+		<label for="firstName" display="hidden" required></label>
+		<input type="text" value="" name="firstName" id="firstName" placeholder="First Name">
+
+		<label for="lastName" display="hidden" required></label>
+		<input type="text" value="" name="lastName" id="lastName" placeholder="Last Name">
+
+		<label for="nickName" display="hidden" required></label>
+		<input type="text" value="" name="nickName" id="nickName" placeholder="Nickname">
+
+		<label for="email" display="hidden" required></label>
+		<input type="text" value="" name="email" id="email" placeholder="Email">
+
+		<label for="avatar"></label>
+		<input id="avatar" name="avatar" type="file"  accept="image/*" value=""/>
+
+		<label for="description" display="hidden"></label>
+		<input type="text" value="" name="description" id="description" placeholder="Description">
+
+
+		<button class="redButton" type="submit">Je démarre l’aventure !</button>
+
+	</form>
+	</section>
+
+ 
+{% endblock %}
+
+
+{% block javascript %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
```

Comment:
Please Do Not Use JQuery for javascript interactions. Change this part of code from JQUERY to JS 
Please put this script into a JS file